### ISSUE TITLE
[Bug] Date editable just showing format string on frontend

### ISSUE THREAD

**Issue by kirjavascript** [pimcore/pimcore#1667]
Using any date editable seems to work completely fine in editmode, but shows the format string on the frontend instead of the actual date.

Build number is 4084

### Expected behavior 

Selected editable date is shown.
  
### Actual behavior  

String for formatting date is shown instead ('F dS Y', 'd m Y' etc)
  
### Steps to reproduce  

Use a date editable with a format string.
  
### Code example 
```php
<?= $this->date("date", [
    "format" => "d m Y"
]); ?>
```

**Comment by weisswurstkanone** [comment]
I believe if Carbon Date is used then the format string must be in this format:http://php.net/manual/de/function.strftime.php

**Comment by kirjavascript** [comment]
Yes, it uses a strftime one on the frontend but a PHP DateTime one on the backend.

Leading you to have to write wrappers like this to normalise the behaviour;

`        if ($this->view->editmode) {
            return $this->view->date($name, [
                 "format" => "F dS Y"
            ]);
        }
        else {
            return $this->view->date($name, [
                 "format" => "%B %e %Y"
            ]);
        }
`
Surely both should use a single format?

**Comment by brusch** [comment]
You can now use `outputFormat` to specify the date for the frontend.

**Comment by langert** [comment]
@brusch do you have an idea how can I get the formated date via the php Api in pimcore x ? 
With 
`$document->getEditable('date')->getDate()`
I can get the carbon date object. But this is noct formated localized and shows 
`2021-08-31 22:00:00`
instead of 
`2021-09-01 00:00:00`
as selected in editmode with format (d.m.Y)

**Comment by NiklasBr** [comment]
@langert try the `->locale()` method, for example:

```php
$document->getEditable('date')->getDate()->locale('de'):
```

https://carbon.nesbot.com/docs/#api-localization

**Comment by langert** [comment]
@NiklasBr unfortunatley same result - missing 2 hours
may this be a bug, based on [https://github.com/pimcore/pimcore/pull/9933](url) ?

**Comment by jdreesen** [comment]
It depends on your server-side timezone

**Comment by langert** [comment]
@jdreesen you are right. Adjuting the the default timezone in php.ini leads to the correct ouput
